tests/Makefile.am

```diff
@@ -1,11 +1,12 @@
 AM_CPPFLAGS = -I$(top_srcdir)/libusb
 LDADD = ../libusb/libusb-1.0.la
-LIBS =
+LDFLAGS = -static
```

Comment:
Maybe worth mentioning in the commit message since it affects all platforms.

Reply:
True. Using static makes it so tests can access things with hidden visibility. Will make that clear.